src/plugins/navigation/navigation.js

```diff
@@ -28,6 +28,12 @@
     // Wait for impress.js to be initialized
     document.addEventListener( "impress:init", function( event ) {
 
+        var root = event.target;
+
+        // Check if actual slide is overview
+        var inOverview = root.querySelector( ".active" ).id === "overview";
```

Comment:
This is not correct.

    id="overview"

Is just another slide and just another id. It may or may not exist, and the presentation author can call it something else than "overview".

If we think more generically of what you're trying to do here, you could consider creating a new plugin called navigation-key-shortcuts (or something shorter, maybe just "shortcuts"). This plugin could allow each step in a presentation to be tied to a keyboard key shortcut. For example:

    <div id="overview" class="step" data-shortcut-key="o" data-x=...>

Reply:
I've added a conditional to taht navigation option, and used impress-overview as the ID. The shorcut key is interesting. I'll try to add it.

Reply:
Is there a way to get the full area of the canvas, to make overview dynamic?

Reply:
While we don't have a more generic implementation, I think this overview could work